lib/core/src/Cardano/Wallet.hs

```diff
@@ -751,6 +754,34 @@ estimateTxFee ctx wid recipients = do
   where
     tl = ctx ^. transactionLayer @t @k
 
+
+-- | Estimate a Byron wallet migration cost by estimating fee by doing migration
+-- coin selection first
+estimateByronWalletMigrationCost
```

Comment:
We shouldn't be talking about `Byron` here, this module is completely agnostic to different kind of wallet. So, let's simply call this one `estimateWalletMigrationCost` :+1:

Reply:
done